src/sys/signal.rs

```diff
@@ -490,11 +490,13 @@ pub enum SigevNotify {
 
 /// Used to request asynchronous notification of the completion of certain
 /// events, such as POSIX AIO and timers.
+#[cfg(not(target_os = "openbsd"))]
```

Comment:
As far as I can tell it's not supported at all. No mention of sigevent in the man pages or [headers](https://github.com/openbsd/src/blob/941122a81b5d8c4d77e8a37913bd2e7d25e3de4b/sys/sys/signal.h).